app/views/unattended/provisioning_templates/snippet/redhat_register.erb

```diff
@@ -108,17 +108,9 @@ description: |
   echo
   echo "Starting the subscription-manager registration process"
 
-  <% if !atomic %>
```

Comment:
I'd prefer to merge https://github.com/theforeman/foreman/pull/9823 first.

Reply:
Yeah, I agree. 
I needed to create this draft PR mostly for the rpm file, so I can use it to test it downstream. 

It just occurred to me that I can rebase this PR onto https://github.com/theforeman/foreman/pull/9823.

Reply:
https://github.com/theforeman/foreman/pull/9823 has been merged